Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.yml

```diff
@@ -932,6 +989,68 @@ script:
     - contextPath: Nexpose.Scan.Status
       description: The scan status. Valid values are aborted, unknown, running, finished, stopped, error, paused, dispatched, integrating
       type: string
+  - name: nexpose-download-report
+    arguments:
+    - name: report_id
+      required: true
+      description: The identifier of the report.
+    - name: instance_id
+      required: true
+      description: The identifier of the report instance. Also supports the "latest"
+        keyword.
+    - name: name
+      description: The report name.
+    - name: format
+      auto: PREDEFINED
+      predefined:
+      - pdf
+      - rtf
+      - xml
+      - html
+      - text
```

Comment:
switch to upper case

Reply:
The original commands use these values and I wanted to be consistent. Should I change it anyway?

Reply:
leave it as is